cMake/FreeCAD_Helpers/SetupPython.cmake

```diff
@@ -3,18 +3,7 @@ macro(SetupPython)
 
     find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
 
-    # For backwards compatibility with old CMake scripts
-    set(PYTHON_EXECUTABLE ${Python3_EXECUTABLE})
```

Comment:
Actually if I bring back this line `set(PYTHON_EXECUTABLE ${Python3_EXECUTABLE})` it starts to work again